AppCenter/AppCenter/MSAppCenter.m

```diff
@@ -286,12 +283,12 @@ - (NSArray *)sortServices:(NSArray<Class> *)services {
   }
 }
 
-- (BOOL)startService:(Class)clazz
-       withAppSecret:(NSString *)appSecret
-          andSendLog:(BOOL)sendLog
-     fromApplication:(BOOL)fromApplication {
+- (BOOL)startService:(Class)clazz andSendLog:(BOOL)sendLog fromApplication:(BOOL)fromApplication {
   @synchronized(self) {
 
+    // If it is started from library, it shouldn't use app secret.
```

Comment:
Nit. Just to be consistent we should do the same with the target token. Since library can't start with a token either.